austation/code/modules/mining/machine_vending.dm

```diff
@@ -0,0 +1,11 @@
+/obj/machinery/mineral/equipment_vendor
+	var/list/ignore_list = list(
+		"Minebot Upgrade: A.I.",
+		"Mining Bot Companion"
+	)
```

Comment:
```suggestion
	var/static/list/ignore_list = list(
		"Minebot Upgrade: A.I.",
		"Mining Bot Companion"
		)
```

Reply:
static lists use one list across all object instances, it uses less memory

Reply:
Thanks. I will try to remember to do this in the future